src/extension.ts

```diff
@@ -26,7 +28,20 @@ export function activate(context: vscode.ExtensionContext) {
         nisabaOutputChannel.append(str + '\n');
     });
 
-    context.subscriptions.push(disposable);
+    const disposable2 = vscode.commands.registerCommand('ucl-rsdg.validationButton', () => {
+        // The code you place here will be executed every time your command is 
+        const firstLine = vscode.window.activeTextEditor.document.lineAt(0);
+        const lastLine = vscode.window.activeTextEditor.document.lineAt(vscode.window.activeTextEditor.document.lineCount - 1);
+        const textRange = new vscode.Range(firstLine.range.start, lastLine.range.end);
+        const filePath = vscode.window.activeTextEditor.document.uri.fsPath;
+        const fileProject = "cams/gkab";
+        const fileContent = vscode.window.activeTextEditor.document.getText(textRange);
```

Comment:
From looking at the [API documentation](https://code.visualstudio.com/api/references/vscode-api), perhaps we don't need to create the `textRange` if we need to get the whole text, but can instead call `vscode.window.activeTextEditor.document.getText()` (with no arguments). That would simplify things a bit. As written, we can easily change which lines we're getting, but I don't think we're likely to need the flexibility.